src/pages/Pool/PositionPage.tsx

```diff
@@ -346,7 +346,32 @@ const useInverter = ({
   }
 }
 
-export function PositionPage() {
+const INVALID_STATE = (
+  <PageWrapper>
+    <div style={{ display: 'flex', alignItems: 'center', flexDirection: 'column' }}>
+      <ThemedText.HeadlineLarge style={{ marginBottom: '8px' }}>
+        <Trans>Position unavailable</Trans>
+      </ThemedText.HeadlineLarge>
+      <ThemedText.BodyPrimary style={{ marginBottom: '32px' }}>
+        <Trans>To view a position, you must be connected to the network it belongs to.</Trans>
+      </ThemedText.BodyPrimary>
+      <PositionPageButtonPrimary as={Link} to="/pools" width="fit-content">
+        <Trans>Back to Pools</Trans>
+      </PositionPageButtonPrimary>
+    </div>
+  </PageWrapper>
+)
+
+export default function PositionPageWrapper() {
```

Comment:
`PositionPage.tsx` should export the `PositionPage`, so could you rename these?
`PositionPageWrapper` -> `PositionPage`
`PositionPage` -> `PositionPageContent`